tldw_Server_API/tests/CI/test_e2e_inprocess_client_contract.py

```
@@ -3,6 +3,8 @@
 
 def test_inprocess_testclient_fallback_enters_lifespan() -> None:
     source = Path("tldw_Server_API/tests/e2e/fixtures.py").read_text(encoding="utf-8")
+    if "from tldw_Server_API.app.main import app" not in source:
+        raise AssertionError("fallback helper must import the shared FastAPI app")
```

Ruff (0.15.7)

[warning] Avoid specifying long messages outside the exception class

(TRY003)